compel/include/uapi/infect.h

```diff
@@ -209,4 +209,12 @@ static inline int parasite_setup_shstk(struct parasite_ctl *ctl,
 #define parasite_setup_shstk parasite_setup_shstk
 #endif
 
+#ifndef compel_gcs_enabled
+static inline bool compel_gcs_enabled(struct user_gcs *gcs)
```

Comment:
It looks like we don't need neither compel_gcs_enabled() nor compel_shstk_enabled(). 
The both only appear in arch/ code and it seems better to just define no underscore version in compel/arch/<arch>/src/lib/include/uapi/asm/infect-types.h  and drop the global stubs.

Reply:
absolutely, I'll do this change as a separate PR for x86 to remove `compel_shstk_enabled`

I have removed `compel_gcs_enabled` in this PR.